robocop/checkers/naming.py

```diff
@@ -143,7 +148,10 @@ def check_keyword_naming(self, keyword_name, node):  # noqa
         if keyword_name.startswith('...'):
             self.report("not-enough-whitespace-after-newline-marker", node=node)
             return
-        if normalize_robot_name(keyword_name) == 'runkeywordif':
+        normalized_name = normalize_robot_name(keyword_name)
+        if normalized_name != keyword_name.replace(' ', '').lower():
```

Comment:
how about simple 
```
if '_' in keyword_name:
```

Reply:
People sometimes like to complicate easy things 😉